src/ui/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
+node_modules
+/build
+/.svelte-kit
+/package
+.env
+.env.*
+!.env.example
+dist/
```

Comment:
no newline

Reply:
no newline at end of file?

Reply:
Yeah it's being stripped by vscode for gitignore files, think they are treated weird at least by my ide setup

Reply:
Approved for now, but you should look into being a normal person one day